types/block.go

```diff
@@ -817,6 +961,51 @@ func (sh SignedHeader) StringIndented(indent string) string {
 		indent)
 }
 
+// ToProto converts SignedHeader to protobuf
+func (sh *SignedHeader) ToProto() *tmproto.SignedHeader {
+	if sh == nil {
```

Comment:
is that even possible?

Reply:
in usage, it may not be possible but while testing it was an issue so I decided to include it